herddb-jdbc/pom.xml

```diff
@@ -238,6 +246,17 @@
                                 </transformer>
                             </transformers>
                             <filters>
+                                <filter>
+                                    <artifact>com.google.guava:guava</artifact>
+                                    <includes>
```

Comment:
can need an IT test to ensure it still runs, a very simple "use calcite" test but otherwise it can be fragile with upgrades IMHO

Reply:
this is probably what you are looking for
https://github.com/diennea/herddb/blob/master/herddb-thirdparty/openjpa-test/pom.xml#L52

Reply:
Can be, generally speaking I try to test in the same module to avoid unrelated surprises ("if openjpa fails it means openjpa module has a bug, not the jdbc shade"). But ok for me

Reply:
I don't know how to test this artifact within the same module, do you mean with the maven-invoker-plugin ?
it would be kind of hard. I have never done this kind of tests before.
Do you have any example ?

Otherwise as you said, we can stay with current codebase, leveraging that module as a sort of integration test

Reply:
For example or just a failsafe/surefire binding on integration-test. @Test can just do a list in target to match the .jar (ensure to ignore -sources/-javadoc jars which happen in releases) and run it with a custom classloader for example.
Can be done later if too hard for this PR.